src/main/java/io/cryostat/rules/RuleProcessor.java

```diff
@@ -231,6 +231,8 @@ public synchronized void accept(TargetDiscoveryEvent tde) {
             case LOST:
                 deactivate(null, tde.getServiceRef());
                 break;
+            case MODIFIED:
```

Comment:
Hmm... I would expect that Rule starting would be broken since there is nothing in this case now, but I just tried it and it worked fine... something I am missing?

Reply:
For MODIFIED state, I am guessing the ServiceRef does not change so it should not affect Rule?

Reply:
Make sense thanks :D